web/src/logic/protocol.ts

```diff
@@ -11,11 +11,24 @@ const PLUGIN_ABI = [
     "function metadataProvider() external view returns (uint256 providerType, bytes location)"
 ]
 
+export const getManager = async() => {
+    const provider = await getProvider()
+    const registryInfo = {
+        address: "0xb4Dc1B282706aB473cdD1b9899c57baD2BD3e2f3",
```

Comment:
Test registry address: `0x9EFbBcAD12034BC310581B9837D545A951761F5A`
https://github.com/5afe/safe-core-protocol/blob/a1b0da08b251b1f7aa6d1ae09b4fc6b474813979/deployments.ts#L1054

Reply:
I will update this in a follow up PR